.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,134 @@
+
+**Note:** If you have not signed the [Magento Contributor Agreement](https://magento.com/content/magento-contributor-agreement), the PR provides a link. We require a signed form and agreement to the terms for contribution.
+
+## Contribution guidelines
+
+Write content using [kramdown](https://kramdown.gettalong.org/), which is a simple markup language. We use kramdown, Liquid, and [Jekyll](https://jekyllrb.com/) to generate a static site. Check [Glossary Template](https://github.com/jcalcaben/gatsby-glossary-app/wiki/Glossary-Template) for a term template and options.
+
+You can update existing or add new terms in the terms directory.
+
+The following guidelines may answer most of your questions and help you get started:
+
+1. Check [existing pull requests](https://github.com/magento/devdocs/pulls) and make sure you are not duplicating work!
+1. For large contributions or changes that include multiple files, [open an issue](#report-an-issue) and discuss it with us first. This may further prevent duplicate or unnecessary effort.
+1. [Chunk many small/medium changes](#tips-for-writing-content) into one or two PRs. This helps us to efficiently and effectively facilitate your contribution.
+1. Familiarize yourself with existing [DevDocs](https://devdocs.magento.com) and [User Guide](https://magento.com/technical-resources) documentation. Look through and search the guides for potential terms and additional information to provide.
+
+## Tips for writing content
+
+Focus on the content with useful information, code samples, and important notes for your fellow Magento developers and community members. Don't forget to review your work for typos, formatting errors, or sentences that need clarifying before opening a pull request.
+
+Use the following guidelines to help you with the writing process:
+
+- Definte the term. What would be helpful for the community and developers?
+- Consider acronyms and synonyms. These help locate and define terms.
+- Keep your sentences concise.
+- Batch several small changes into a single pull request (instead of separate ones) to ensure your contributions get approved and merged quickly. Have several typo fixes across several terms? Batch them into one PR.
+- Remember to use active voice (not passive), write in the present tense, and use a friendly tone in second person. For example, _"The log captures commands, output..."_.
+- Provide links to additional documentation and information if appropriate.
+- Need help with format? See our [Glossary template](https://github.com/jcalcaben/gatsby-glossary-app/wiki/Glossary-Template).
+
+### Edit metadata
+
+The Markdown (.md) file's metadata is a set of YAML key-value pairs. The metadata section is located at the top of each file and required to correctly label and add your term to the glossary. For more info, see the [Glossary template](https://github.com/jcalcaben/gatsby-glossary-app/wiki/Glossary-Template).
```

Comment:
How about adding a note that ALL metadata tags need to be listed, even if they contain no value. Also an example of what a property that has no value would help.